### DISCUSSION TITLE
Want to enable public discussions without categories

### DISCUSSION THREAD

**Discussion by paritshivani** [Sunbird-Ed/Community#626] [Sunbird ED configuration discussion] [answered]
Hello Team,
Hope you are doing well!..

In our project we have few custom requirements as follows
1. Client want discussions groups and discussions to be public. It means they want all users can see all available groups and can request to join the group
2. Once we enable the discussions, and open the forum we can see one default category named as "Discussion forum for groups". We don't want this category wise segregation of discussion threads. we want all threads should visible on that category page only and menu should be renamed as "all" instead of categories.

My questions are
1. Can we make groups public?
2. can we remove category wise segregation of discussion?
3. If not, Where I can find the UI code for the menus and category card? so that i can make changes

@vinukumar-vs , @santhosh-tg , @arunpilli21

**Comment by vinukumar-vs** [comment]
@paritshivani 

I don't understand what is mean by groups public. If we don't know which user is requesting(public), how can you add user to group.
Think through the solution & share the more details.
Today any logged in user can see all the groups & can join by accepting the terms & conditions.
> 1. Can we make groups public?

Yes. You can remove the category wise segregation. Please refer the below discussion-ui library configuration to enable & disable the menu configurations.
> 2. can we remove category wise segregation of discussion?

If the option is not meeting your requirement, please feel free to contribute the required changes to the below discussion-ui library 
https://github.com/Sunbird-Lern/discussions-UI/

@AmiableAnil @Krishnaj20 @ashokreddy1208

**Reply by Krishnaj20** [reply, marked as answer]
@paritshivani - current implementation of groups doesn't allow for public discovery on the lines of whatsapp groups. However, if there is a need to have groups public, you can extend groups to implement for such a use case. I would request you to go thru the below documentation and ask specific follow up queries for community to be able to provide assistance.
Detailed documentation about groups can be found here: https://lern.sunbird.org/learn/product-and-developer-guide/groups
Detailed documentation about discussion forums can be found here: https://lern.sunbird.org/learn/product-and-developer-guide/discussion-forum

@AmiableAnil - Pls add if anything is missing.

**Comment by ashokreddy1208** [comment]
@paritshivani  let us know if there is any additional help needed from the community.

**Comment by ashokreddy1208** [comment]
@paritshivani any update on this issue? If the issue is resolved then can you mark this thread as answered

**Reply by Krishnaj20** [reply]
@paritshivani - closing this thread as there has been no response from your side. Feel free to start a new thread if you have any follow up questions.
